trajopt_optimizers/trajopt_optimizers_test_suite/CMakeLists.txt

```diff
@@ -0,0 +1,97 @@
+cmake_minimum_required(VERSION 3.5.0)
+project(trajopt_optimizers_test_suite)
+
+find_package(console_bridge REQUIRED)
+# find_package(Pagmo REQUIRED)
+find_package(OsqpEigen REQUIRED)
+find_package(trajopt_ifopt REQUIRED)
+find_package(trajopt_utils REQUIRED)
+# find_package(trajopt_pagmo REQUIRED)
+find_package(trajopt_sqp REQUIRED)
+find_package(ifopt REQUIRED)
+find_package(Eigen3 REQUIRED)
+find_package(GTest REQUIRED)
+
+###########
+## Build ##
+###########
+
+add_library(${PROJECT_NAME} SHARED
+    # src/basic_optimization_unit.cpp
+    # src/cart_position_optimization_unit.cpp
+    src/joint_position_optimization_unit.cpp
+    src/joint_velocity_optimization_unit.cpp
+    include/basic_optimization_unit.hpp
+    # include/cart_position_optimization_unit.hpp
+    include/joint_position_optimization_unit.hpp
+    include/joint_velocity_optimization_unit.hpp
+)
+target_include_directories(${PROJECT_NAME}
+  PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:include>
+)
+target_link_libraries(${PROJECT_NAME} PUBLIC
+    ifopt::ifopt_core
+    console_bridge
+    # Pagmo::pagmo
+    OsqpEigen::OsqpEigen
+)
+trajopt_target_compile_options(${PROJECT_NAME} PUBLIC)
+trajopt_clang_tidy(${PROJECT_NAME})
+list (APPEND PACKAGE_LIBRARIES ${PROJECT_NAME})
+
+
+#############
+## Install ##
+#############
+
+# trajopt_configure_package(${PACKAGE_LIBRARIES})
```

Comment:
You'll need to include this macro. It is what installs the targets. If building with catkin_tools, turn on the install space and make sure it works with `catkin config --install` then rebuild.

Reply:
You'll need to make a cmake.in file like shown [here](https://github.com/ros-industrial-consortium/trajopt_ros/blob/master/trajopt_ifopt/cmake/trajopt_ifopt-config.cmake.in)